src/__tests__/spectral.test.ts

```diff
@@ -15,6 +15,7 @@ describe('spectral', () => {
     expect(results.length).toBeGreaterThan(0);
   });
 
+  // Assures: https://stoplightio.atlassian.net/browse/SL-786
```

Comment:
I like this assures comment, could be good to add to our best practices

Reply:
Yeah except this is intended to be a community project, and folks outside stoplight can't access that URL. Perhaps replace with a brief summary of the thing it assures instead of the link?